generate.ml

```diff
@@ -621,16 +631,17 @@ let gen_random_function ast =
      prototype: If the list of formals is empty, the argument list will be
      (void) instead of empty (). *)
   begin match f with
-  | GFun (fdec, _) -> Cil.setFormals fdec fdec.sformals
-  | _ -> ()
+    | GFun (fdec, _) -> Cil.setFormals fdec fdec.sformals
+    | _ -> ()
```

Comment:
Could you undo this bit please? My usual style doesn't indent the `|` markers.

Reply:
Will do.